torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -18,6 +18,44 @@ struct TORCH_API L1LossOptions {
 
 // ============================================================================
 
+/// Options for a KLDiv loss module.
+struct TORCH_API KLDivLossOptions {
+  KLDivLossOptions(Reduction::Reduction reduction = Reduction::Mean)
+      : reduction_(reduction) {}
+
+  /// Specifies the reduction to apply to the output.
+  TORCH_ARG(Reduction::Reduction, reduction);
+};
+
+// ============================================================================
+
+/// Options for a MSE loss module.
+struct TORCH_API MSELossOptions {
+  MSELossOptions(Reduction::Reduction reduction = Reduction::Mean)
+      : reduction_(reduction) {}
+
+  /// Specifies the reduction to apply to the output.
+  TORCH_ARG(Reduction::Reduction, reduction);
+};
+
+// ============================================================================
+
+/// Options for a BCE loss module.
+struct TORCH_API BCELossOptions {
+  BCELossOptions(
+      Tensor weight = {},
+      Reduction::Reduction reduction = Reduction::Mean)
+      : weight_(weight), reduction_(reduction) {}
```

Comment:
I think as a convention we should only provide a non-default constructor when the options has non-optional arguments or when the options has only one argument. For `BCELossOptions` I think we can follow the design of `HingeEmbeddingLossOptions` by providing defaults to `weight` and `reduction` and removing the non-default constructor.